SpringProject/src/test/java/com/gradproject2019/ConferenceServiceTest.java

```diff
@@ -23,7 +26,7 @@
 public class ConferenceServiceTest {
 
     @InjectMocks
-    private ConferenceServiceImpl conferenceService;
+    private ConferenceServiceImpl conferenceServiceImpl;
```

Comment:
This should be of type 'ConferenceService', shouldn't it?

Reply:
when I make it a `ConferenceService` I get this error: `Cannot instantiate @InjectMocks field named 'conferenceService'! Cause: the type 'ConferenceService' is an interface.`